### ISSUE TITLE
[Bug]: cmd+F shortcut to search not working on latest build

### ISSUE THREAD

**Issue by ishaanrathodd** [th-ch/youtube-music#2781]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a bug report that matches the one I want to file, without success.
- [X] I understand that **th-ch/youtube-music has NO affiliation with Google or YouTube**

### YouTube Music (Application) Version

v3.7.1

### Checklists

- [X] I use the portable version of the YouTube Music Application.
- [ ] I can reproduce this issue in the [official version of (WEB) YTM](https://music.youtube.com).

### What operating system are you using?

macOS

### Operating System Version

macOS sequoia 15.2 beta

### What CPU architecture are you using?

arm64 (including Apple Silicon)

### Last Known Working YouTube Music (Application) version

v3.6.2

### Reproduction steps

just press cmf+F in the app to activate search, wont work now.

### Expected Behavior

should work

### Actual Behavior

doesn't work

### Enabled plugins

Shortcuts, Synced lyrics, Navigation, Video Toggle Native

**Comment by JellyBrick** [comment]
Can you try disable `Shortcuts`?

**Comment by ishaanrathodd** [comment]
> Can you try disable `Shortcuts`?

yes, i disabled and tried & it didn't work, i re-enabled it again and it didn't work either
